claat/cmd/export.go

```diff
@@ -106,8 +106,8 @@ func exportCodelab(src string, opts CmdExportOptions) (*types.Meta, error) {
 	// codelab export context
 	lastmod := types.ContextTime(clab.mod)
 	meta := &clab.Meta
```

Comment:
maybe more general (in case it ever needs to be used elsewhere) to do this:

clab.Meta.src = src

and then anything that uses clab.Meta will have access to the orig src doc.

Reply:
Done :)